components/company-name.jsx

```diff
@@ -13,6 +13,7 @@ export function CompanyName ({
 	placeholder = 'Please enter your company name',
 	isRequired = true,
 	isHidden = false,
+	maxlength = '',
```

Comment:
This should be a number.
```suggestion
	maxlength = 40,
```
?

Reply:
nope,i think  default can be  empty as i dont want to force the limit on all the consumers of this components

Reply:
I agree with Ludo on this. If Agne wants the limit to be 40, then there's a reason for this and we should be consistent across all of the consumers.

Reply:
that has been resolved by setting it default to 100